app/src/main/java/org/onionshare/android/ShareManager.kt

```diff
@@ -166,7 +168,10 @@ class ShareManager @Inject constructor(
         sharing: ShareUiState.Sharing,
     ) = withContext(Dispatchers.IO) {
         when (state) {
-            WebserverManager.State.STARTED -> _shareState.value = sharing
+            WebserverManager.State.STARTED -> {
+                _shareState.value = sharing
+                notificationManager.onSharing()
```

Comment:
Unfortunately I can't properly test this because Tor is exiting repeatedly on my test phone, even with the LocalSocket timeout set to 0. :-/